packages/nocodb/src/lib/meta/helpers/webhookHelpers.ts

```diff
@@ -268,16 +267,11 @@ export function _transformSubmittedFormDataForEmail(
   // @ts-ignore
   formView,
   // @ts-ignore
-  columns: Column[]
+  columns: Record<string, any>[]
```

Comment:
The type could be ColumnType.

Reply:
`columns` includes `Column[]` and some extra fields.`ColumnType` shouldn't be used as well.

Reply:
Then you can do `(Column &  { additionalTypes : Type })[]`